test/passing/ite_fit_or_vertical.ml.ref

```diff
@@ -0,0 +1,137 @@
+let _ = if b then e else ( e1 ; e2 )
+
+let _ =
+  if b then
+    e
+  else (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more )
+
+let _ =
+  if b then (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more )
+  else if b1 then (
+    something loooooooooooooooooooooooooooooooong enough to_trigger a break ;
+    this is more )
+  else
+    e
+
+;;
+f
+  ( if loooooooooooooooooooooooooooooooooooooooooooooooooooooooooong then
+      ()
+    else
+      () )
```

Comment:
Here, it looks like indentation starts from the point before `if` rather than the point before `(`. Is this correct, and intended? If so, it would be good to move that into a separate option. There are other cases where indentation starts from before leading parens (and I currently consider it a bug when this doesn't happen), and while it's expected for opinions to vary, we should aim to work toward a point where each value of the option is consistently obeyed.

Reply:
The indentation starting from the `if` was precisely the point of the last commit, but I can revert it, ship the PR without it (and keep the current indentation behavior), and we can put the changes of this last commit in another PR indeed.